entry/src/main/js/default/i18n/music_info.json

```diff
@@ -0,0 +1,8 @@
+[
+  {"music_cover_image":"/common/images/dreams_img.png","music_name":"Dream","music_genre":"Romantic"}

```

Comment:
Check to use the the media apis / from widget from SDK to get the actual music details played. And if no music is playing, indicate the same.